tools/vargo/src/main.rs

```diff
@@ -1186,6 +1250,41 @@ fn run() -> Result<(), String> {
                 }
             }
 
+            #[cfg(target_os = "macos")]
+            {
+                let macos_prepare_script_path =
+                    target_verus_dir.join("set_permissions_and_allow_gatekeeper.sh");
+                std::fs::write(&macos_prepare_script_path, macos_prepare_script)
+                    .map_err(|x| format!("could not write to macos prepare script ({})", x))?;
+                std::fs::set_permissions(
+                    &macos_prepare_script_path,
+                    <std::fs::Permissions as std::os::unix::fs::PermissionsExt>::from_mode(0o755),
+                )
+                .map_err(|x| {
+                    format!("could not set permissions on macos prepare script ({})", x)
+                })?;
+            }
+
+            #[cfg(target_os = "linux")]
+            {
+                let linux_prepare_script_path = target_verus_dir.join("set_permissions.sh");
+                std::fs::write(&linux_prepare_script_path, linux_prepare_script)
+                    .map_err(|x| format!("could not write to linux prepare script ({})", x))?;
+                std::fs::set_permissions(
+                    &linux_prepare_script_path,
+                    <std::fs::Permissions as std::os::unix::fs::PermissionsExt>::from_mode(0o755),
+                )
+                .map_err(|x| {
+                    format!("could not set permissions on linux prepare script ({})", x)
+                })?;
+            }
+
+            if let Some(version_info) = verus_version {
+                let version_info_path = target_verus_dir.join("version.txt");
+                std::fs::write(&version_info_path, version_info.version)
+                    .map_err(|x| format!("could not write to version file ({})", x))?;
+            }
+
```

Comment:
Is this the reason that the version-writing commands are commented out from the CI portion?

Reply:
Because I couldn't get the windows one to work, and they are redundant with this.